src/Bicep.Core.Samples/Files/baselines/ResourceDerivedTypes_LF/main.formatted.bicep

```diff
@@ -34,11 +32,15 @@ param bar resource<'Microsoft.Resources/tags@2022-09-01'> = {
   }
 }
 
-output baz resource<'Microsoft.ManagedIdentity/userAssignedIdentities@2023-01-31'> = {
+output baz resource<
+  'Microsoft.ManagedIdentity/userAssignedIdentities@2023-01-31'
+> = {
```

Comment:
The formatting seems a bit awkward to me...

@anthony-c-martin @jeskew would you have any concerns if I add a special case to prevent line breaks between `resource<` and `>`?

Reply:
No objections from me! This is what the pretty printer did by default, and I thought it made sense since C# lets you split type arguments to generics over multiple lines. We may want to allow something similar if we introduce generics at some point, but I agree that `resource<x>` is easiest to read on a single line, even if it's longer than the max line length.